samples/dotnet-azurefunction/README.md

```diff
@@ -316,7 +319,50 @@ Given differnt secret store, the metadata string needs to be provided. In order
 [DaprSecret("vault", "my-secret",  Metadata = "metadata.version_id=15&metadata.version_stage=AAA"`.
 ```
 
-# Step 4 - Cleanup
+## 5. Dapr Invoke output binding:
+Dapr invoke output binding is could be used to invoke other azure functions or service where it will act as a proxy. For example, In the below Azure function, which gets triggered on HttpTrigger, can invoke antother azure functions like RetrieveOrder.
```

Comment:
> where it will act as a proxy.

Do you think this will be confusing? Also, invoke output binding can only be used to invoke Azure Functions registered with Dapr Service Invocation trigger, should be call that out?

```suggestion
Dapr invoke output binding can be used to invoke other Azure functions or services where it will act as a proxy. For example, In the below Azure function, which gets triggered on HttpTrigger, can invoke another Azure functions like RetrieveOrder.
```